content/docs/tasks_usage/traffic_management/iptables_redirection.md

```diff
@@ -217,7 +217,7 @@ The following snippet from the demo `curl` client's init container spec shows th
 Init Containers:
   osm-init:
     Container ID:  containerd://80f86af7bc64b7a70f7f2bf64242d735d857559a79cd97e206513368130902f1
-    Image:         openservicemesh/init:v0.8.0
+    Image:         openservicemesh/init:v0.9.0
```

Comment:
This is probably also a spot we don't need to update for every release. No issue with including it here though.

Reply:
thank you @nojnhuh